### ISSUE TITLE
Color Contrast: Onboarding: The bright grey progress bar part on the white fails contrast ratio

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77332]
## Action Performed:
Prerequisites:
- the user has just created a new account

1. Open the Expensify app
2. Start the onboarding flow
3. Measure the color contrast ratio between the light grey part of the progress bar and the white background

## Expected Result:
The visual presentation of the graphical objects should have a contrast ratio of at least 3:1 against the adjacent color.

## Actual Result:
The light grey color of the progress bar #E7E2DC on the white background #FCFBF9 fails the minimum requirements with the color contrast ratio of 1.24:1

## Area issue was found in:
Onboarding

## Failed WCAG checkpoints
WCAG 1.4.11 Non-text Contrast
https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast

## User impact:
People with low vision often have difficulty perceiving graphics that have insufficient contrast. This can be exacerbated if the person has a color vision deficiency that lowers the contrast even further. Providing a relative luminance (lightness difference) of 3:1 or greater can make these items more distinguishable when the person does not see a full range of colors.

## Suggested resolution:
Insufficient contrast can make it difficult for individuals with low vision or color blindness to read text on a screen. To improve accessibility, use the darker shade of the focus indicator. For example, the color #A28E7C provides the 3.03:1 color contrast ratio against the same background.
https://www.w3.org/TR/WCAG21/#non-text-contrast

## Environment(s)
Samsung Galaxy S24 Ultra - Android 14, Partner - Native (No mobile browser)
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos
<img width="1547" height="863" alt="Image" src="https://github.com/user-attachments/assets/1d671002-b369-4f48-8563-fd272f673328" />

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7005558
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by rushatgabhane** [comment]
This issue was retested in **BFV cycle 544085 (Native Apps)** on 2026-04-28 and is **Not Fixed**.

### Screenshots / Videos
- https://platform.applause.com/services/links/v1/external/c8c5a2f51063dd2de1e04cd68af7b5b713d291e742477fb4f00cec38a4c396ae

### Retest notes
The issue is not fixed

Description:The light grey color of the progress bar (#E6E1DA) on the light background (#FCFBF9) fails the minimum requirements with the color contrast ratio of 1.3:1.

Issue verified on Apple iPhone 16 - iOS 26.2.1, Wi-Fi - Native
App Version: Version 9.3.63 and App Name: Expensify
Please see the attached screenshot

**Comment by dubielzyk-expensify** [comment]
This is not a crucial element and doesn't need to pass accessibility requirements.
